test/worlds/plowing_effect_test.world

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+  <world name="default">
+    <!-- Gravity -->
+    <gravity>4.0 0 -9.75</gravity>
+    <!-- plowing effect ground plane -->
+    <include>
+      <uri>model://plowing_effect_ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://sun</uri>
+    </include>
+
+    <model name="sphere1">
+      <link name='base_link'>
+        <pose>0 0 0 0 0 0</pose>
+        <inertial>
+          <mass>0.5</mass>
+          <inertia>
+            <ixx>0.22799999999999998</ixx>
+            <iyy>0.7435210984814149</iyy>
+            <izz>0.9655210984814149</izz>
```

Comment:
I would expect the `ixx`, `iyy` and `izz` values to be equal for a sphere

Reply:
Thanks, I have corrected it.